cmd/datamon/cmd/bundle_list.go

```diff
@@ -7,14 +7,16 @@ import (
 	"text/template"
 
 	"github.com/oneconcern/datamon/pkg/core"
+	"github.com/oneconcern/datamon/pkg/model"
 
 	"github.com/spf13/cobra"
 )
 
+// BundleListCommand describes the CLI command for listing bundles
 var BundleListCommand = &cobra.Command{
 	Use:   "list",
 	Short: "List bundles",
-	Long:  "List the bundles in a repo",
+	Long:  "List the bundles in a repo, ordered by their key",
```

Comment:
ordered by their bundle ID. There is no concept of keys in the CLI model for datamon, that is internal.